sessionctx/variable/sysvar.go

```diff
@@ -69,8 +69,23 @@ const (
 	BoolOff = "OFF"
 	// BoolOn is the canonical string representation of a boolean true.
 	BoolOn = "ON"
+
+	// On is the canonical string for ON
+	On = "ON"
+
+	// Off is the canonical string for OFF
+	Off = "OFF"
+
 	// Nightly indicate the nightly version.
 	Nightly = "NIGHTLY"
+	// Warn means return warnings
+	Warn = "WARN"
+	// OffInt is used by TiDBMultiStatementMode
+	OffInt = 0
```

Comment:
I would suggest you mark a type to `OffInt` and the field `TiDBMultiStatementMode` itself,  like `type TiDBMultiStatementModeEnum int` or something else. Then move it closer to the variable field or the handle function.

Reply:
OK, I'll move it.